parser/src/parser/grammar.lalrpop

```diff
@@ -155,6 +155,67 @@ RandElem: RandBinding = {
     <name: Identifier> <size: Size> => RandBinding::new(name, size)
 }
 
+// EVALUATOR FUNCTIONS
+// ================================================================================================
+
+EvaluatorFunction: EvaluatorFunction = {
+    "ev" <evaluator_fn_name: Identifier>
+        "(" <main_cols: MainCols?> <cols_separator: (",")?> <aux_cols: AuxCols?> ")" ":"
```

Comment:
Could we simplify this to the following? Then we could remove the error checking for the `cols_separator` to simplify the error handling below as well.

```
"(" <main_cols: MainCols?> "," <aux_cols: AuxCols?> ")" ":"
```

Reply:
Hanging comma seems a little weird to me personally. (e.g. `(main: [a, b, c], )` or `(, aux: [a, b, c])`. But if you think it's worth it to keep the grammar cleaner, I can change it.

Reply:
Another benefit of adjusting in this way is that you can use `IntegrityStmts` for the `integrity_constraints` parsing so that the error handling isn't duplicated

Reply:
Thank you! This is much cleaner. Fixed.